src/movepick.cpp

```diff
@@ -99,13 +99,15 @@ void MovePicker::score() {
 
   static_assert(Type == CAPTURES || Type == QUIETS || Type == EVASIONS, "Wrong type");
 
-  for (auto& m : *this)
+  for (ExtMove *p = cur; p < endMoves; ++p)
```

Comment:
I'm curious about what is the cause of the speedup here. Is it the changed for loop?